LibraBFT/Impl/Consensus/ChainedBFT/RoundManager/Properties.agda

```diff
@@ -16,16 +16,16 @@ open import LibraBFT.Impl.Base.Types
 open import LibraBFT.Impl.Consensus.Types
 open import LibraBFT.Impl.Util.Util
 
-module LibraBFT.Impl.Consensus.ChainedBFT.EventProcessor.Properties
```

Comment:
This should also be moved to `LibraBFT.Impl.Consensus.RoundManager.Properties`, keeping the structure consistent with the implementation.

Reply:
DONE